scrapers/DorcelVision.yml

```
@@ -10,34 +10,65 @@ movieByURL:
     url:
       - https://www.dorcelvision.com/en/movies
     scraper: movieScraper
+sceneByName:
+  action: scrapeXPath
+  queryURL: https://www.dorcelvision.com/en/search?type=4&keyword={}
+  scraper: sceneSearch
+sceneByQueryFragment:
+  action: scrapeXPath
+  queryURL: "{url}"
+  scraper: sceneScraper
+
 
 xPathScrapers:
+  sceneSearch:
+    common:
+      $card: //div[@id="entriesList"]/div
+    scene:
+      Title: $card/details/div/h3
+      URL: 
+        selector: $card/a[@class="movies rolloverv2"]/@href
+        postProcess: 
+          - replace:
+            - regex: (.*)
+              with: https://www.dorcelvision.com$1
+              
+      Image: 
+        selector: $card/a[@class="movies rolloverv2"]/img[last()]/@data-src
+        postProcess: 
+          - replace:
+              - regex: (.*)
+                with: https://www.dorcelvision.com$1
+      Details: $card/details/summary
```

Some of these have longer descriptions, try this:
```suggestion
      Details: $card/details//p[last()]
```